Extension/src/LanguageServer/Providers/codeActionProvider.ts

```diff
@@ -214,6 +218,26 @@ export class CodeActionProvider implements vscode.CodeActionProvider {
                 } else {
                     return;
                 }
+            } else if (command.command === "C_Cpp.ExtractToFunction" ||
+                command.command === "C_Cpp.ExtractToMemberFunction") {
+                if (!hasExperimentalFeatures) {
+                    return;
+                }
+                if (command.arguments && command.arguments.length === 1) {
+                    disabledReason = command.arguments[0];
```

Comment:
Can we add a `disabledReason` on the JSON object instead of hacking `arguments` to mean something other than the arguments that we pass to the code action?

Reply:
sure, I'll check

Reply:
done